src/main/java/lotto/LottoMachine.java

```diff
@@ -0,0 +1,26 @@
+package lotto;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LottoMachine {
+    private Randoms randoms;
+
+    public LottoMachine(){
+
+    }
+
+    public ArrayList<Lotto> playLotto(int count) {
+        ArrayList<Lotto> lottoLists = new ArrayList<>();

```

Comment:
초기 크기를 설정해주면 기본 크기 보다 커졌을 때 생성되는 불필요한 메모리 복사를 막을 수 있어요! 
그리고 객체의 타입은 추상화된 객체로 처리하는 것이 더 적절해보여요
```suggestion
        List<Lotto> lottoLists = new ArrayList<>(count);

```